pages/governance/proposing_a_new_market.md

```diff
@@ -35,10 +35,10 @@ The following decribes how to set various parameters for a new market and assume
 - Reference price (e.g. `40,000`) and `p` value
   - Defines the exponent on the reference price as `p:=FLOOR(log10(reference_price))`. For example: `p:=FLOOR(log10(40,000))=4`
 - Exchanges required for oracles (`exchange_config_json`)
-- Liquidity Tier (these are based on the liquidity tiers recommended, so mappings could change if these tiers change)
+- Liquidity Tier (can be updated through governance vote at a later date.)
   - 0: Large Cap (BTC/ETH)
-  - 1: Mid-Cap
-  - 2: Long-tail
+  - 1: Mid-Cap (Markets with at least 8 robust oracle sources with liquidity >= 50K on both sides and 30d daily spot trading volume >= $100M.)
```

Comment:
Think it depends on how conservative the recommendation for new markets should be. There are some markets with 7 oracles (APE and SEI) that are tier 2 because they are more recently launched. I think it might make sense for new markets to be recommended at long-tail unless considered safe.